tests/test_spark.py

```diff
@@ -318,6 +318,53 @@ def test_secrets():
         LOGGER.warn("Error when deleting secret, {}".format(r.content))
 
 
+# Skip DC/OS < 1.10, because it doesn't have support for file-based secrets.
+@pytest.mark.skip("Enable when SPARK-22131 is merged and released in DC/OS Spark")
+@pytest.mark.skipif('shakedown.dcos_version_less_than("1.10")')
+@pytest.mark.sanity
+@pytest.mark.tls
+def test_driver_executor_tls():
+    '''
+    Put keystore and truststore as secrets in DC/OS secret store.
+    Run SparkPi job with TLS enabled, referencing those secrets.
+    '''
+    shakedown.run_dcos_command('package install --cli dcos-enterprise-cli --yes')
+    resources_folder = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), 'resources'
+    )
+    keystore_file = 'server.jks'
+    truststore_file = 'trust.jks'
+    keystore_path = os.path.join(resources_folder, '{}.base64'.format(keystore_file))
+    truststore_path = os.path.join(resources_folder, '{}.base64'.format(truststore_file))
+    keystore_secret = '__dcos_base64__keystore'
+    truststore_secret = '__dcos_base64__truststore'
+    shakedown.run_dcos_command('security secrets create /{} --value-file {}'.format(keystore_secret, keystore_path))
```

Comment:
how come this works now? I remember having to jump through some hoops to get the secrets API to work (https://github.com/mesosphere/spark-build/blob/master/tests/utils.py#L179). Could you homogenize the way secrets are added in the tests by either using the SecretsHandler or changing/removing it?